test/e2e/scripts/couchdb3-cluster.yml

```
@@ -0,0 +1,58 @@
+version: '3.9'
+
+services:
+  couchdb-1.local:
+    image: public.ecr.aws/s5s3h4s7/cht-couchdb:4.4.0
+    volumes:
+      - ${DB1_DATA}:/opt/couchdb/data
+      - cht-credentials:/opt/couchdb/etc/local.d/
+    environment:
+      - "COUCHDB_USER=${COUCHDB_USER:-admin}"
+      - "COUCHDB_PASSWORD=${COUCHDB_PASSWORD:?COUCHDB_PASSWORD must be set}"
+      - "COUCHDB_SECRET=${COUCHDB_SECRET}"
+      - "COUCHDB_UUID=${COUCHDB_UUID}"
+      - "SVC_NAME=${SVC1_NAME:-couchdb-1.local}"
+      - "CLUSTER_PEER_IPS=couchdb-2.local,couchdb-3.local"
+      - "COUCHDB_LOG_LEVEL=${COUCHDB_LOG_LEVEL:-log}"
+    ports:
+      - "${COUCH_PORT}:5984"
+      - "${COUCH_CLUSTER_PORT}:5986"
```

Minor: Since it can be run in uncontrolled environments I'd prefer to be more defensive
```suggestion
      - "127.0.0.1:${COUCH_PORT}:5984"
      - "127.0.0.1:${COUCH_CLUSTER_PORT}:5986"
```

---

These are just e2e tests. Supposedly only devs run them.

---

To clarify I mean things like a dev machine running without a firewall on a public wifi, but the actual data that could be accessed is not sensitive afaik so I'm not overly concerned